client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -393,16 +393,23 @@ const Audit = ({ electionId }: { electionId: string }) => {
 
   if (!election.isSuccess) return null
 
-  const { auditName, auditType, jurisdictions, rounds } = election.data
+  const { id, auditName, auditType, jurisdictions, rounds } = election.data
 
   return (
     <Column>
       <H2>{auditName}</H2>
-      <Tag large style={{ marginBottom: '15px' }}>
+      <div style={{ marginBottom: '10px' }}>
+        <AnchorButton href={`/api/support/elections/${id}/login`} icon="log-in">
+          Log in as audit admin
+        </AnchorButton>
+      </div>
+      <Tag large style={{ marginBottom: '10px' }}>
```

Comment:
I feel like I want the audit type tag to stay next to the audit name to make it clear that it's a property of that audit.

What do you think about putting them on the same line like so?
<img width="570" alt="Screen Shot 2022-09-07 at 11 00 46 AM" src="https://user-images.githubusercontent.com/530106/188948404-a297223f-51d4-4548-a6d6-b0dca27f303e.png">

Reply:
(I also added `intent="primary"` to the log in button, just an idea)

Reply:
Nice - I like both those changes!